envoy/network/filter.h

```diff
@@ -330,6 +330,20 @@ class ListenerFilter {
    * @return status used by the filter manager to manage further filter iteration.
    */
   virtual FilterStatus onAccept(ListenerFilterCallbacks& cb) PURE;
+
+  /**
+   * Called when data read from the connection. If the filter chain doesn't get
+   * enough data, the filter chain can be stopped, then waiting for more data.
+   * @param buffer the buffer of data.
+   * @return status used by the filter manager to manage further filter iteration.
+   */
+  virtual FilterStatus onData(Buffer::Instance& buffer) PURE;
+
+  /**
+   * Return the size of data the filter want to inspect from the connection.
+   @return the size of data inspect from the connection. 0 means filter needn't any data.
+   */
+  virtual size_t inspectSize() { return 0; }
```

Comment:
rename to `maxReadBytes()`

Reply:
got it

Reply:
sorry, forget update, will update next around